config/deploy/prod_server.rb.template

```diff
@@ -1,44 +0,0 @@
-set :application, "nucore"
```

Comment:
Are we removing this template because there is sufficient customization for each school that a template doesn't make sense? Or is this supplanted by other files?